core/impl-base/src/main/java/org/jboss/arquillian/core/impl/ManagerImpl.java

```diff
@@ -597,6 +599,27 @@ void debug(ObserverMethod method, boolean interceptor)
       }
    }
 
+   private void debugExtension(Class<?> extension)
```

Comment:
I believe we should do it here. It is related to this change and will not result in any noise in my personal opinion. I am not sure if we should really follow such a granularity that every refactoring / boyscouting has to be resolved as separate issue and PR.